src/parse/rules/go_rules.build_defs

```diff
@@ -549,10 +549,10 @@ def go_get(name:str, get:str|list, repo:str='', deps:list=[], exported_deps:list
     if binary:
         outs = ['bin/' + name]
     else:
+        outs = [f'pkg/{CONFIG.OS}_{CONFIG.ARCH}/{out}' for out in outs]
         # Outputs are created one directory down from where we want them.
         # For most it doesn't matter but the top-level one will get lost.
-        cmd += [' if [ -f ${OUTS}.a ]; then mkdir -p $OUTS && mv ${OUTS}.a $OUTS; fi']
-        outs = [f'pkg/{CONFIG.OS}_{CONFIG.ARCH}/{out}' for out in outs]
+        cmd += [f' if [ -f {out}.a ]; then mkdir -p {out} && mv {out}.a {out}; fi' for out in outs]
```

Comment:
Can we through single quotes around `{out}.a` so that filenames with spaces aren't broken?

```python
         cmd += [f' if [ -f "{out}.a" ]; then mkdir -p "{out}" && mv "{out}.a" "{out}"; fi' for out in outs] 
```

Reply:
> Thanks for doing this. I thought it'd be that simple after walking through this in the issue, but I didn't want to own writing a test to validate that this doesn't get broken in the future. Cheers.

yea the tests are a little more complex than the fix. Still thinking about how best to do them. Also good point about the quotes will modify when I create the pr for the tests

Reply:
@sean- we actually have been having this problem with the space in filenames. For example, if a repository is inside of a directory(or any parent directory) with space in names, the build doesn't seem to work. The obvious solution would be having quotes on all file names, however, it can introduce other issues as well.

Reply:
The use of a shell script as the intermediate target for build rules is clever, but also problematic at the same time.  I don't have a better solution to using quotes everywhere (and escaping the values that are later quoted, which also seems to be a missing function call).

One possibility is to change to an accumulated list of argv arrays that are passed to `cmd.Exec()` or some such.  That would be a big refactor, but ultimately probably the right thing to do.  And a PITA for everyone.

I don't have answers here.  We looked into using quotes and found it introduced other subtle problems and punted.  ¯\_(ツ)_/¯